### DISCUSSION TITLE
SEARCHING LINUX USER

### DISCUSSION THREAD

**Discussion by raphael069** [Len-Stevens/Python-Antivirus#14] [Q&A]
### **hello,**
If you see this and are a Linux user pls contact me. I´m currently thinking about a new "feature" but cant test it on Linux.
 `my contact details (just write me a Mail); `
therealcookie@waifu.club
_(nice Mail i know xD)_

**Comment by Len-Stevens** [comment]
I am on Linux, we can chat here

**Reply by raphael069** [reply]
Thats amazing I want to add a feature so when you download the Antivirus it will download the hashes like normal but in a zip file to save to reduce download time but I´m not sure if you can use or unpack zip files with Python on linux systems since zip is a windows format?

**Reply by raphael069** [reply]
Here I made a beta can you test it? Tell me when there are any errors.
[AV_beta.zip](https://github.com/Len-Stevens/Python-Antivirus/files/8864650/AV_beta.zip)

**Reply by MandiYang** [reply]
@cookie0o 
Hello, I tested the beta version on my Ubuntu 22.04 LTS, python3.10 and the script doesn't seem to really work. 

A little bit of debug output:
```
WARNING: QApplication was not created in the main() thread.
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
```

![Screenshot from 2022-12-03 17-01-40](https://user-images.githubusercontent.com/64307609/205450160-7c349288-fc62-4c13-9758-f11ad8035cb1.png)




![Screenshot from 2022-12-03 17-00-19](https://user-images.githubusercontent.com/64307609/205450095-8c1148b2-511e-4ee0-bfc8-d4c3018954ed.png)

**If needing anymore information reply to me**

**Reply by raphael069** [reply]
I think it's because of threading I'm currently working on version 3.0 try to use a version from the repo in my profile they are a bit more up to date and thanks for your feedback
